controller.js

```diff
@@ -47,10 +47,16 @@ exports.getUsers = (req, res) => {
   });
 };
 
-exports.getArticles = (req, res) => {
-  fetchArticles().then((articles) => {
-    res.status(200).send({ articles });
-  });
+exports.getArticles = (req, res, next) => {
+  const { sort_by } = req.query;
+  const { order } = req.query;
+  const { topic } = req.query;
```

Comment:
Keep your code DRY by destructuring in the same variable, ie. `const {sort_by, order, topic}` since they will all be coming from req.query anyway :)